examples/complete/main.tf

```diff
@@ -37,3 +37,71 @@ module "create_scc_instance" {
   en_instance_crn                   = module.event_notification.crn
   skip_cos_iam_authorization_policy = false
 }
+
+module "create_scc_controls" {
+  source                      = "../../controls/."
+  instance_id                 = module.create_scc_instance.guid
+  control_library_name        = "control_library_name"
+  control_library_description = "control_library_description"
+  control_library_type        = "custom"
+  latest                      = true
+  version_group_label         = "de38e8c4-2212-4e4b-8dcf-b021b98d8e43"
```

Comment:
where did you get this from? Is this another thing we need to ask for a data lookup for?

Reply:
yes, i think so. think i just pulled it from an instance i created on ui to test example.